src/IO/InterfileHeaderSiemens.cxx

```diff
@@ -226,9 +226,9 @@ InterfileRawDataHeaderSiemens::InterfileRawDataHeaderSiemens()
   add_key("tracer activity at time of injection (bq)", KeyArgument::NONE, &KeyParser::do_nothing);
   add_key("injected volume (ml)", KeyArgument::NONE, &KeyParser::do_nothing);
   add_key("horizontal bed translation", KeyArgument::NONE, &KeyParser::do_nothing);
-  add_key("start horizontal bed position (mm)", KeyArgument::NONE, &KeyParser::do_nothing);
+  add_key("start horizontal bed position (mm)", &bed_position_horizontal);
```

Comment:
I'm fairly sure that Siemens keywords are different from the standard Interfile ones (check one of your files). If not, just remove these lines, else remove the normal one, and add the new one (pointing to the same field).

Reply:
Looking at a Siemens header they're the same, so I'll delete.